customTypes/wix-data.d.ts

```diff
@@ -0,0 +1,603 @@
+declare module "wix-data" {
```

Comment:
too long for me. how do you know that those definitions work? we need to understand how we can test it..